src/targets/gpu/CMakeLists.txt

```diff
@@ -237,6 +238,19 @@ else()
     endif()
 endif()
 
+if(NOT WIN32)
+    include(MigxLocalTarget)
+    rocm_local_targets_migx(MIGXGPU_TARGET)
+    message(STATUS "Compiling for gfx arch: ${MIGXGPU_TARGET}")
+
+    foreach(agent IN LISTS MIGXGPU_TARGET)
+        list(APPEND offload_arch_flags "--offload-arch=${agent}")
```

Comment:
Wont this conflict with `GPU_TARGETS`?

Reply:
what's the difference between AMDGPU_TARGETS or GPU_TARGETS ?

Reply:
`AMDGPU_TARGETS` is used by many math libraries, but its deprecated and undocumented. The documented variable is `GPU_TARGETS`.